blocks/editor-media-upload/index.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * WordPress dependencies
+ */
+import { select } from '@wordpress/data';
+import { mediaUpload } from '@wordpress/utils';
+
+export default function editorMediaUpload( filesList, onFileChange, allowedType ) {
```

Comment:
Let's add a doc comment to this function since it's exported.

Reply:
a0043246